js/adapt-stateful-session.js

```diff
@@ -177,7 +179,9 @@ export default class StatefulSession extends Backbone.Controller {
     // If responseType doesn't contain any data, assume that the question
     // component hasn't been set up for cmi.interaction tracking
     if (_.isEmpty(responseType)) return;
-    const id = `${this.scorm.getInteractionCount()}-${questionModel.get('_id')}`;
+    const id = this._uniqueInteractionIds
+      ? `${this.scorm.getInteractionCount()}-${questionModel.get('_id')}`
+      : questionModel.get('_id');
```

Comment:
Should all `_uniqueInteractionIds` logic be in https://github.com/adaptlearning/adapt-contrib-spoor/blob/master/js/scorm/wrapper.js instead?

Reply:
No, cos it's not a setting for the wrapper, it's a setting for the recordInteractions function in the session.